app/assets/javascripts/host_edit.js

```diff
@@ -146,13 +146,10 @@ function update_capabilities(capabilities) {
 var stop_pooling;
 
 function submit_with_all_params() {
-  var url = $('form').attr('action');
-  if (url.match('hostgroups')) {
-    resource = 'hostgroup';
-  } else {
-    resource = 'host';
-  }
-  capitalized_resource = resource[0].toUpperCase + resource.slice(1);
+  var url = $('form').attr('action'),
+  resource = 'host',
```

Comment:
I am missing the statement witch would decide the resourse. Aren't we missing this later on?
```js
if (url.match('hostgroups')) resource = 'hostgroup';
```

Reply:
Removed `data-submit => progress_bar` on this [line](https://github.com/theforeman/foreman/pull/4783/commits/46190888489a020c58e2f6f495f095a522e1b38a#diff-db336c69095130c90178c09661f8e4cfR2) for hostgroup that's why here it is not checking for `hostgroup`.

Reply:
ah, understood, but than we do not need any branching depending on the resource. We can assume 'host' being the only option.

Reply:
Updated changes in separate commit